pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_unpublishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
+[id="unpublishing-a-module-in-pantheon_{context}"]
+= Unpublishing a module in {ProductShortName}
+
+Unpublish a module in {ProductShortName} from the Customer Portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to unpublish.
++
+If you don't see the module listed, search for the modules using the search box.
```

Comment:
So you have to know the module's name? There's no browse functionality? Is the search an exact match, or will it provide results if the search term is approximate?

Reply:
Looks like this PR covers searching - https://github.com/redhataccess/pantheon/pull/225

I don't know whether to include details about searching here. How about adding a link to searching modules in additional resources?

Reply:
Added a placeholder to link to Searching Modules under Additional Resources

Reply:
I don't see this in 0a83c34

Reply:
It's there though - https://github.com/redhataccess/pantheon/pull/254/files#diff-df1e50fb25e7b80b7f88efcccc77722fR30

Reply:
LGTM